site/content/documentation/tutorials/getting-started.md

```diff
@@ -384,44 +399,39 @@ We have more sophisticated options at our disposal, however.
 {{< example "Example 09" "model/Example09Filter.java" >}} shows how we can use {{< javadoc "Model.filter" "model/Model.html#filter-org.eclipse.rdf4j.model.Resource-org.eclipse.rdf4j.model.IRI-org.eclipse.rdf4j.model.Value-org.eclipse.rdf4j.model.Resource...-" >}} to "zoom in" on a specific subject in our model. We're also using the opportunity to show how you can print out RDF statements in a slightly prettier way:
 
 ```java {linenos=inline}
-ValueFactory vf = SimpleValueFactory.getInstance();
-
-// We want to find all information about the artist `ex:VanGogh` in our model
-IRI vanGogh = vf.createIRI("http://example.org/VanGogh");
+// We want to find all information about the artist `ex:VanGogh`.
+IRI vanGogh = iri("http://example.org/VanGogh");
```

Comment:
Here it would be very helpful if we use `Values.iri(...)` for users who copy paste from this example.